src/components/notifications/CallMeBot.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="mb-3">
+        <label for="callmebot-endpoint" class="form-label">{{ $t("Endpoint") }}</label>
+        <input id="callmebot-endpoint" v-model="$parent.notification.callMeBotEndpoint" type="text" class="form-control" required>
+        <i18n-t tag="div" keypath="callMeBotGet" class="form-text">
+            <a href="https://www.callmebot.com/blog/free-api-facebook-messenger/" target="_blank">Facebook Messenger</a>
+            <a href="https://www.callmebot.com/blog/test-whatsapp-api/" target="_blank">WhatsApp</a>
+            <a href="https://www.callmebot.com/blog/telegram-phone-call-using-your-browser/" target="_blank">Telegram Call</a>
+            <!--There is currently no documentation available. I contacted the support for further information.-->
```

Comment:
What was the response here?
I'd not include this comment, as it is implied that the linked docs are best-effort.

```suggestion
```

Reply:
I'm actually being ghosted :/
I'll do some testing and add it there

Reply:
The rate limits are approx.:
- WhatsApp: 1 message per 10 sec
- Telegram Call: 1 message per 65 sec

Another interesting thing is, that sometimes callmebot combines messages:  
![Screenshot](https://github.com/louislam/uptime-kuma/assets/50715457/fed0b8c0-6a9a-4539-b9b4-06a2915fe5bd)

Reply:
I added my results in: 88caa31
Displayed as:
![image](https://github.com/louislam/uptime-kuma/assets/50715457/84aae2c7-5e42-4aa0-93ca-d2bcc5147b06)